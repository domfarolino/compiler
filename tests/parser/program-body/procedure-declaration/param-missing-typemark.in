program dominic is
  procedure domFunc(typelessArg1 [10:20]);
begin end program.
