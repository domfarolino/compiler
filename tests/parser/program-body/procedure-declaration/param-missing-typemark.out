[1;31mLine 2: Expected ')' after parameter list in procedure header, but got: 'typelessArg1'
Line 2: Error parsing program body
[0m